torch/_inductor/codecache.py

```diff
@@ -346,7 +346,7 @@ def get_path(
 def get_hash(content: Union[str, bytes], extra: str = "", hash_type: str = "code"):
     if hash_type == "code":
         return code_hash(content, extra)
-    if hash_type == "cubin":
+    if hash_type == "cubin" or "hsaco":
```

Comment:
Sorry, but `hash_type == "cubin" or "hsaco"` is equivalent to `hash_type == "cubin"`, isn't it?
```suggestion
    if hash_type in ["cubin", "hsaco"]:
```